### PR TITLE
Updating alt text for splash pages

### PR DESCRIPTION
**Pull request by ahmad-shahid**

### CODE REVIEW

File: src/gcintranet/splash.ejs [hunk 1]
```diff
@@ -18,7 +18,11 @@
             <h1 property="name" class="wb-inv">GC Intranet / Intranet du GC</h1>
             <div class="row">
                 <div class="col-xs-11 col-md-8">
-                    <img src="<%- wet.builder.environment(pr.cdnEnv) %>wet-boew/assets/sig-spl.svg" width="283" alt="Government of Canada / Gouvernement du Canada">
+                    <img src="<%- wet.builder.environment(pr.cdnEnv) %>wet-boew/assets/sig-spl.svg" width="283" alt="Government of Canada">
```
Comment (plup2) [inline src/gcintranet/splash.ejs:21]: This doesn't have anything to do with that new image Pierre was referring to this morning right? (just checking)

Comment (ahmad-shahid) [reply src/gcintranet/splash.ejs:21]: Yes, that's right - not related to that :)

**Review by plup2 — APPROVED** [review]

File: src/gcweb/splash.ejs [hunk 2]
```diff
@@ -18,7 +18,11 @@
             <h1 property="name" class="wb-inv">Canada.ca</h1>
             <div class="row">
                 <div class="col-xs-11 col-md-8">
-                    <img src="<%- wet.builder.environment(pr.cdnEnv) %>wet-boew/assets/sig-spl.svg" alt="Government of Canada / Gouvernement du Canada">
+                    <img src="<%- wet.builder.environment(pr.cdnEnv) %>wet-boew/assets/sig-spl.svg" alt="Government of Canada" property="logo" width="283">
+                    <span class="wb-inv"> / <span lang="fr">Gouvernement du Canada</span></span>
+                    <meta property="name" content="Government of Canada">
```
Comment (nschonni) [inline src/gcweb/splash.ejs:23]: The `<meta>` tag isn't intended to be used outside of the `<head>` https://html.spec.whatwg.org/multipage/semantics.html#the-meta-element

Comment (plup2) [reply src/gcweb/splash.ejs:23]: That code conforms to the canada.ca established splash page.

Also, according to MDN,  `<meta name>` is allowed for any element that accept metadata content (like `<script>`, etc)
(while `<meta charset>` and `<meta http-equiv>` being restricted to `<head>`)

(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta)

**Pull request merged**